File: VineyardSite/VineyardSite/Model/Order.cs
namespace VineyardSite.Model;

public class Order
{
    public Guid Id { get; set; }
    public ICollection<OrderItem> OrderItems { get; set; }
    
    public double TotalPrice { get; set; }
    public DateTime Date { get; set; }
    public string Address { get; set; }
    public string DeliveryType { get; set; }
    public string PaymentType { get; set; }
    public string Status { get; set; }
    public string Notes { get; set; }
    public string Email { get; set; }
    public User User { get; set; }
    public string UserId { get; set; }
    
}

File: VineyardSite/AngularFrontend/VineyardSiteAngular/src/app/Models/Order.ts
import { IOrderItem } from "./OrderItem";
import { IUser } from "./User";

export interface IOrder {
    Id: string,
    OrderItems: IOrderItem[],
    TotalPrice: number,
    Date: Date,
    Address: string,
    DeliveryType: string,
    PaymentType: string,
    Status: string,
    Notes: string,
    Email: string,
    User: IUser,
    UserId: string
}
